### ISSUE TITLE
Something wrong with selected object.

### ISSUE THREAD

**Issue by ZP-Z** [millag/DiscreteElasticRods#2]
![image](https://user-images.githubusercontent.com/48114681/60954064-f115b480-a330-11e9-9c0c-41cdf36164e1.png)

![image](https://user-images.githubusercontent.com/48114681/60953636-1fdf5b00-a330-11e9-91fa-b6d096e36847.png)
like this. It runs well without a selected object. But it crashed when I selected an object.
Is there any issue with the Object files?
Thanks.

**Comment by ZP-Z** [comment]
![image](https://user-images.githubusercontent.com/48114681/60954406-7f8a3600-a331-11e9-8f15-8e03b17246f4.png)
simulation

**Comment by millag** [comment]
Yes I'm afraid during refactoring this got broken. Will fix it at some point this month. Thanks for reporting!
